### PR TITLE
AG-1072 Fix scores bar chart bug

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

**Comment by sagely1** [comment]
@hallieswan you can just look at the commits for AG-1072

File: src/app/features/charts/components/score-barchart/score-barchart.component.ts [hunk 1]
```diff
@@ -71,19 +71,32 @@ export class ScoreBarChartComponent implements AfterViewInit, OnChanges {
...
+      }
+    });
+  }
+
   initData() {
+    if (!this.data)
+      return;
+    
     if (!this.data)
       return;
```
Comment (hallieswan) [inline src/app/features/charts/components/score-barchart/score-barchart.component.ts]: Could one of these statements be removed?

**Review by hallieswan — APPROVED** [review]
Looks good to me!

File: src/app/features/charts/components/score-barchart/score-barchart.component.ts (hunk 1 above)
Comment (sagely1) [reply src/app/features/charts/components/score-barchart/score-barchart.component.ts]: Removed.  Thanks!

**Pull request merged**